browser/js/bundle.entry.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  bel: require("bel"),
+  morphdom: require('morphdom'),
```

Comment:
Btw what do you mean by copying events? I don't think I'm relying on that anywhere, am I?